ao_quality_control/views/qc_menu.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <menuitem name="Configuration"
+              id="quality_control.menu_qc_config"
+              groups="quality_control.group_quality_control_manager"
+              parent="quality_control.qc_menu"
+              sequence="100"/>
+
+    <menuitem name="Quality Control Teams"
+              id="quality_control_team.qc_team_config_menu"
```

Comment:
@aheficent Do you think there is another way to do this avoiding to use an external id from other module?

Reply:
I don't think so. The only thing is a different menu then.